pinaxcon/registrasion/models.py

```diff
@@ -103,7 +103,7 @@ def clean(self):
         if errors:
             raise ValidationError(dict(errors))
 
-    def save(self):
+    def save(self, force_insert=False, force_update=False, using=None):
```

Comment:
So. this particular change is required in order for the generic object.save() thing to work.